src/devices/Card/Ndef/UriType.cs

```diff
@@ -0,0 +1,235 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Runtime.Serialization;
+using System.Text;
+
+namespace Iot.Device.Ndef
+{
+    /// <summary>
+    /// Uri type
+    /// </summary>
+    public enum UriType
+    {
+        /// <summary>
+        /// No prepending is done, and the URI field
+        /// contains the unabridged URI.
+        /// </summary>
+        [Description("")]
+        NoFormat = 0x00,
+
+        /// <summary>
+        /// http://www.
+        /// </summary>
+        [Description("http://www.")]
+        HttpWww = 0x01,
+
+        /// <summary>
+        /// https://www.
+        /// </summary>
+        [Description("https://www.")]
+        HttpsWww = 0x02,
+
+        /// <summary>
+        /// http://
+        /// </summary>
+        [Description("http://")]
+        Http = 0x03,
+
+        /// <summary>
+        /// https://
+        /// </summary>
+        [Description("https://")]
+        Https = 0x04,
+
+        /// <summary>
+        /// tel:
+        /// </summary>
+        [Description("tel:")]
+        Tel = 0x05,
+
+        /// <summary>
+        /// mailto:
+        /// </summary>
+        [Description("mailto:")]
+        MailTo = 0x06,
+
+        /// <summary>
+        /// ftp://anonymous:anonymous@
+        /// </summary>
+        [Description("ftp://anonymous:anonymous@")]
+        FtpAnonymousAnonymous = 0x07,
+
+        /// <summary>
+        /// ftp://ftp.
+        /// </summary>
+        [Description("ftp://ftp.")]
+        FtpFtp = 0x08,
```

Comment:
Why not just Ftp?

Reply:
to be coherent with the rest, so it gives a better hint that the url is ftp://ftp.